src/parser/jobs/blu/modules/DoTs.tsx

```diff
@@ -72,36 +100,90 @@ export class DoTs extends CoreDoTs {
 			</Trans>,
 			value: this.unbuffedSongsOfTorment,
 		}))
+
+		const unbuffedWhyBoM = <Trans id ="blu.breath_of_magic.suggestion.unbuffed.why">
+			<Plural value={this.unbuffedBoM} one="# Breath of Magic was" other="# Breaths of Magic were"/> cast without both buffs.
+		</Trans>
+		const unbuffedWhyFlame = <Trans id ="blu.mortal_flame.suggestion.unbuffed.why">
+			<Plural value={this.unbuffedMortal} one="# Mortal Flame was" other="# Mortal Flames were"/> cast without both buffs.
+		</Trans>
+		const unbuffedWhyBoth = <Trans id ="blu.big-dots.suggestion.unbuffed.why">
+			<Plural value={this.unbuffedBoM} one="# Breath of Magic" other="# Breaths of Magic"/>,
+			<Plural value={this.unbuffedMortal} one="# Mortal Flame" other="# Mortal Flames"/> were cast without both buffs.
+		</Trans>
+		const unbuffedWhy = (this.unbuffedBoM && this.unbuffedMortal) ? unbuffedWhyBoth
```

Comment:
Rather that putting this into a big set of const assignmens and a ternary selection, can we just make a function that takes in the two `unbuffed<thing>` booleans and spits back out the correct `why`?